dev/com.ibm.ws.org.apache.myfaces.2.3/bnd.bnd

```diff
@@ -84,58 +85,63 @@ Include-Resource: \
-  org.apache.myfaces.view.facelets.tag.jsf.html;thread-context=true;mandatory:="thread-context", \
-  org.apache.myfaces.view.facelets.tag.jstl.core;thread-context=true;mandatory:="thread-context", \
-  org.apache.myfaces.view.facelets.tag.jstl.fn;thread-context=true;mandatory:="thread-context", \
-  org.apache.myfaces.view.facelets.tag.ui;thread-context=true;mandatory:="thread-context", \
-  org.apache.myfaces.view.facelets.util;thread-context=true;mandatory:="thread-context", \
-  org.apache.myfaces.webapp;thread-context=true;mandatory:="thread-context"
+  org.apache.myfaces.*;version=${Implementation-Version}, \
+  org.apache.myfaces.application;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.application.viewstate;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.cdi.model;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.cdi.view;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.cdi.util;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.component;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.component.search;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.component.visit;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.context;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.context.servlet;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.config.annotation;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.config.impl.digester.elements;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.flow;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.flow.cdi;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.lifecycle;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.push.cdi;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.renderkit;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.renderkit.html;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
+  org.apache.myfaces.shared.context.flash;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
```

Comment:
I'm not sure I quite understand why we do `!org.apache.myfaces.renderkit.html` and then ` org.apache.myfaces.renderkit.html;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \`

Reply:
The first `!org.apache.myfaces.renderkit.html` prevents the classes in those packages from being added to the bundle, and the second `org.apache.myfaces.renderkit.html;thread-context=true...` puts the package name and TCCL directive into the bundle's `Export-Package` list.  

This obviously isn't ideal, but my testing has shown that exporting `org.apache.myfaces.renderkit.html;thread-context=true...`  in the `thirdparty.jsf-2.3` bundle prevents that package from being resolved correctly by the host `myfaces.2.3` bundle.  That's because the `thirdparty.jsf-2.3`  is now only providing a small subset of the o.a.m. packages.  The TCCL directives do work in that bundle if the entire set of o.a.m. packages are provided by it.

Reply:
With my latest commit we no longer need to have this duplication - I'll mark this resolved